### ISSUE TITLE
PSNR Behavior on MoCap-Desk2 with Event + RGB / Event-only Training

### ISSUE THREAD

**Issue by gaojianzhen** [knelk/enerf#12]
Hello,
    I encountered an issue while training on the MoCap-Desk2 dataset and would like to ask for clarification regarding the PSNR behavior under different training settings.
 1. Event + RGB Mixed Training (PSNR = inf)
     When I train the model using Event + RGB mixed supervision, the reported PSNR during evaluation becomes inf.
After inspecting the code, it seems that PSNR is being computed on both RGB-view predictions and event-view renderings, although event views do not have a meaningful intensity ground truth.
Is this behavior expected?
Should PSNR be computed only on RGB camera views when using mixed Event + RGB training?
  2. RGB-only PSNR Saturation (~22.5 [dB)]

<img width="471" height="99" alt="Image" src="https://github.com/user-attachments/assets/eedee270-8884-4a06-ac27-89c696727e34" />
<img width="531" height="308" alt="Image" src="https://github.com/user-attachments/assets/dd2507dc-7b1e-4074-bd40-f1bfdf88516b" />

 After modifying the evaluation code to compute PSNR only on RGB camera frames, the PSNR stabilizes at around 22.5 dB on MoCap-Desk2.
 However, even as training continues and the number of epochs increases, the PSNR does not improve further.
 My questions are:
    Is ~22.5 dB the expected or final PSNR for MoCap-Desk2 under this setting?
   3. Event-only Training (Low PSNR ~12 dB)
When training the model using event-only supervision, the reported PSNR is around 12.x dB.
Is PSNR a meaningful metric for event-only NeRF training, given that events do not directly represent absolute intensity?
Is such a low PSNR expected in the event-only setting？
